src/azul/plugins/repository/tdr/__init__.py

```diff
@@ -131,7 +131,33 @@ def _bigquery(self) -> bigquery.Client:
             return bigquery.Client(project=self._source.project)
 
     def _run_sql(self, query: str) -> BigQueryRows:
-        return self._bigquery.query(query)
+        attempt = 0
+        delays = (10, 20, 40, 80)
```

Comment:
some of the weirdness can be addressed if this is prepended with a 0.

Reply:
added place holder of 0 in the first value, removes need increment by 1 in some locations.

Reply:
Moving back to an equation that calculates the delay can simply the code more.

Reply:
I prefer sticking with the tuple and reworking the control flow instead.